src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+function click(tab, selectedTabId, onTabSelected) {
+  if (tab.id !== selectedTabId) {
+    onTabSelected(tab);
+  }
+}
```

Comment:
Should put the function inside the `Tabs` component
selectedTabId, onTabSelected will be available inside the component
```suggestion
const handleTabSelect = (tab) => {
  if (tab.id !== selectedTabId) {
    onTabSelected(tab);
  }
}
```